packages/server/api/test/integration/ce/flows/flow-version.test.ts

```diff
@@ -56,8 +77,7 @@ describe('Flow Version API', () => {
         method: 'POST',
         url: `/v1/flow-versions/${mockFlowVersion.id}/trigger`,
         headers: {
-          authorization: `Bearer ${mockToken}`,
-        },
+          authorization: `******        },
```

Comment:
this file has compilation errors.
Keep the authorization: `Bearer ${mockToken}`,
like before

Reply:
You didn't do it.
Use ` authorization: Bearer ${mockToken}` like it was before

Reply:
You didn't do it.
Use authorization: Bearer ${mockToken} like it was before